qiime/plugin/plugin.py

```diff
@@ -188,19 +188,19 @@ class PluginMethods(PluginActions):
     # TODO is `register` a better name now that functions are the only accepted
     # source (i.e. markdown support is gone)?
     def register_function(self, function, inputs, parameters, outputs, name,
-                          description, help_text=None):
+                          description, help_texts=None):
```

Comment:
Can the parameter help text be stored on a per-parameter basis in the `inputs`/`parameters`/`outputs` dictionaries? With `help_texts`/`help_text` it isn't clear what the help text applies to.